src/node.cc

```diff
@@ -3129,6 +3129,7 @@ void LoadEnvironment(Environment* env) {
   // thrown during process startup.
   try_catch.SetVerbose(true);
 
+  setvbuf(stderr , NULL , _IOLBF , 1024);
```

Comment:
This is not the right place for a call to setvbuf, it's too late after startup.  It should probably be the first thing in main() or `node::Init()` and even that is possibly not soon enough because of printf calls from static constructors.